test/services/editions/db/EditionsDBTest.scala

```diff
@@ -0,0 +1,143 @@
+package services.editions.db
+
+import java.time._
+
+import com.gu.pandomainauth.model.User
+import fixtures.{EditionsDBService, UsesDatabase}
+import model.editions._
+import org.scalatest.{FreeSpec, Matchers, OptionValues}
+
+class EditionsDBTest extends FreeSpec with Matchers with EditionsDBService with OptionValues {
+
+  "The editions DB" - {
+    "should insert an empty issue" taggedAs UsesDatabase in {
+      val skeleton = EditionsIssueSkeleton(
+        ZonedDateTime.of(2019, 9, 30, 0, 0, 0, 0, ZoneId.of("Europe/London")),
+        ZoneId.of("Europe/London"),
+        Nil
+      )
+      val now = OffsetDateTime.of(2019, 7, 16, 17, 23, 23, 0, ZoneOffset.ofHours(1))
+      val id = editionsDB.insertIssue("daily-edition",
+        skeleton,
+        User("Billy", "Bragg", "billy.bragg@justice.example.com", None),
+        now
+      )
+
+      val retrievedIssue = editionsDB.getIssue(id).value
+      retrievedIssue.displayName shouldBe "daily-edition"
+      retrievedIssue.createdEmail shouldBe "billy.bragg@justice.example.com"
+      retrievedIssue.createdOn shouldBe now.toInstant.toEpochMilli
+      retrievedIssue.createdBy shouldBe "Billy Bragg"
+      val issueDate = OffsetDateTime.ofInstant(Instant.ofEpochMilli(retrievedIssue.issueDate), ZoneId.of(retrievedIssue.timezoneId))
+      issueDate.getDayOfWeek shouldBe DayOfWeek.MONDAY
+      retrievedIssue.launchedOn.isDefined shouldBe false
+      retrievedIssue.launchedBy.isDefined shouldBe false
+      retrievedIssue.launchedEmail.isDefined shouldBe false
+      retrievedIssue.fronts shouldBe Nil
+    }
+
+    "should list the empty issue" taggedAs UsesDatabase in {
+      val issues = editionsDB.listIssues("daily-edition", LocalDate.of(2019, 9, 29), LocalDate.of(2019, 10, 3))
```

Comment:
Does this test depend on the previous one? Could they be independent?

Reply:
It does. It could. If we want them to be independent then we should redo evolutions each time to enforce this, which is very possible.

Reply:
I've isolated the tests by running the evolutions each time (and rolling them back).